README.md

```diff
@@ -678,7 +678,7 @@ Table of Contents
   * [ImprovMX](https://improvmx.com) – Free email forwarding
   * [inumbo.com](http://inumbo.com/) — SMTP based spam filter, free for 10 users
   * [kickbox.io](https://kickbox.io/) — Verify 100 emails free, real-time API available
-  * [mailazy.com](https://mailazy.com/) — 250 emails/day free
+  * [mailazy.com](https://mailazy.com/) — 500 emails/day free, 50,000 emails/month under the beta program!
```

Comment:
not sure if we should add this tbh, we specifically exclude betas as a matching criteria for the list - so beta features is likewise not going to be important (and we'll not know to update this list when the beta ends)

Reply:
@ripienaar thank you for reviewing it. I have removed the beta program offering. could you please review it again? thank you!